.github/workflows/pull-request.yml

```
@@ -24,6 +24,54 @@ env:
+        uses: actions/setup-node@v6
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: 🔄 Fetch main
+        run: git fetch origin main:refs/remotes/origin/main
+
+      - name: 🦋 Generate changeset
+        run: node ./scripts/generate-renovate-changeset.js
+
+      - name: 💾 Commit generated changeset
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)